packages/plugin-ext/src/plugin/tree/tree-views.ts

```diff
@@ -341,10 +342,11 @@ class TreeViewExtImpl<T> implements Disposable {
 
                 // Ask data provider for a tree item for the value
                 // Data provider must return theia.TreeItem
-                const treeItem: TreeItem2 = await this.treeDataProvider.getTreeItem(value);
+                const treeItem: TreeItem = await this.treeDataProvider.getTreeItem(value);
```

Comment:
Nitpick: This type can be inferred.
```suggestion
                const treeItem = await this.treeDataProvider.getTreeItem(value);
```